apis/datadoghq/common/const.go

```diff
@@ -56,6 +56,8 @@ const (
 	DefaultDogstatsdPort = 8125
 	// DefaultDogstatsdPortName default dogstatsd port name
 	DefaultDogstatsdPortName = "dogstatsd"
```

Comment:
The default dogstatsd port name `dogstatsd` is different from the `DogstatsdHostPortName`, which is `dogstatsdport` [(line 200)](https://github.com/DataDog/datadog-operator/pull/841/files#diff-9088a8abc4fac93cee6e4dd256cffc750221214fc8cf62040974e443b9f9853fR200). Should they be made the same? I.e., both be `dogstatsdport` like in the [helm chart](https://github.com/DataDog/helm-charts/blob/57a411047e49e9e31cad8532effb3d7e0e9b3ce3/charts/datadog/templates/agent-services.yaml#L84C13-L84C26).

Reply:
i'd be in favor of this